.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "python.pythonPath": "venv/bin/python",
+  "python.pythonPath": "/cshome/yuntian1/python/bin/python",
```

Comment:
Revert this line, you should always use virtualenv to develop your project anyway